CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+## [1.0.5] - 2016-09-23
```

Comment:
You are right, I should have followed semver a bit better ;). I will try to bring my PR up to date with your latest changes and follow semver numbering.